Utils/build/lib/dataset/dataset.py

```diff
@@ -0,0 +1,74 @@
+from collections import namedtuple
+from typing import List
+
+annotation = namedtuple('annotation', 'tag, start, end, ne')
+
+
+class Dataset:
+    def __init__(self):
+        self.dataset = []  # храним набор tuple (text, [(annotation, start, end, ne)])
+        self.crop_sentences = []
+
+    def load_sentences(self, ann_file_path: str, text_file_path: str):
+        with open(text_file_path, 'r') as text_file:
+            text = text_file.read()
+
+        with open(ann_file_path, 'r') as ann_file:
+            annotations = []
+            for line in ann_file.readlines():
+                tokens = line.split()
```

Comment:
How many `tokens` does a line contain? 

Instead of directly indexes, we can pass an array like a tuple: 

`tokens = tuple(line.split())`

`annotations.append(annotation(*(tokens[1:]))` or `annotations.append(annotation._make(tokens[1:]))`